QuizApp/Scenes/QuizPage/ViewController/QuestionView.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  QuestionView.swift
+//  QuizApp
+//
+//  Created by Tamuna Kakhidze on 15.11.24.
+//
+
+import UIKit
+
+// MARK: - ColorfulBackgroundView
+final class QuestionView: UIView {
+    
+    // MARK: - UI Components
+    private let questionView: UITextView = {
+        let textView = UITextView()
+        textView.translatesAutoresizingMaskIntoConstraints = false
+        textView.font = .systemFont(
+            ofSize: FontSizes.med14,
+            weight: .medium
+        )
+        textView.textColor = CustomColors.neutralDarkGrey
+        textView.textAlignment = .center
+        textView.isEditable = false
+        textView.textContainerInset = .zero
+        textView.backgroundColor = .clear
```

Comment:
Default value should already be nil which is transparent

Reply:
it's white